use-shopping-cart/src/reducers.js

```diff
@@ -48,25 +48,59 @@ export const cartValuesInitialState = {
   totalPrice: 0,
   cartCount: 0
 }
-function Entry(productData, quantity, currency, language) {
+
+function Entry(
+  product,
+  quantity,
+  currency,
+  language,
+  price_metadata,
+  product_metadata
+) {
   const getProductId = () => {
-    if (productData.id) {
-      return productData.id
+    if (product.id) {
+      return product.id
     }
-    if (productData.price_id) {
-      return productData.price_id
+    if (product.price_id) {
+      return product.price_id
     }
-    if (productData.sku_id) {
-      return productData.sku_id
+    if (product.sku_id) {
+      return product.sku_id
     }
-    if (productData.sku) {
-      return productData.sku
+    if (product.sku) {
+      return product.sku
     }
     return uuidv4()
   }
```

Comment:
Also, just realized this but since we're only using this function in one place and these are all simple conditional statements this should just be one `const` statement utilizing the `||` "or" operator.

```js
const id = product.id || product.price_id || product.sku_id || product.sku || uuidv4()
```